frontend/src/utilities/useWatchImages.tsx

```diff
@@ -14,7 +14,7 @@ export const useWatchImages = (): {
   const [images, setImages] = React.useState<ImageInfo[]>([]);
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```